File: Difficulty: Easy/Second Largest/second-largest.java
class Solution {
    public int getSecondLargest(int[] arr) {
        // code here
        int max = 0;
        int max2 = 0;
        for(int num:arr)
        {
            if(num>max)
            {
                max2 = max;
                max = num;
            }
            else if(num<max && num>max2) max2 = num;
        }
        return max2==0?-1:max2;
    }
}

File: Difficulty: Easy/Second Largest/second-largest.cpp
class Solution {
  public:
    int getSecondLargest(vector<int> &arr) {
        // code here
        int max = -1;
        int secMax = -1;
        int n = arr.size();
        for(int i=0;i<n;i++)
        {
            int temp = arr[i];
            if(temp>max)
            {
                secMax = max;
                max = temp;
            }
            else if(temp>secMax && temp < max)
            {
                secMax = temp;
            }
        }
        return secMax;
        
    }
};